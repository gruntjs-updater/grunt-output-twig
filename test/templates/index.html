{#
 # This is the homepage template!
 #}

{% extends "_layout" %}
{% set title = entry.heading %}

{% block test %}
woo
{% endblock %}

{% block content %}
	<div class="intro">
		{{ entry.siteIntro }}
	</div>
{% endblock %}
