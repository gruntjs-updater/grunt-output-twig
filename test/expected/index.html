<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US">
<head>
			<meta charset="utf-8" />
		<title>On the Rocks</title>
		<meta name="viewport" content="width=1000" />
		<link rel="home" href="" />
		<link rel="stylesheet" type="text/css" href="/resources/css/ontherocks.css" />
	</head>

<body>
	<header id="header">
		<h1><a href="">On the Rocks</a></h1>

		<nav>
			<ul>
				<li><a  href="cocktails">Cocktails</a></li>
				<li><a  href="ingredients">Ingredients</a></li>
				<li><a  href="blog">Blog</a></li>
			</ul>
		</nav>

		<ul id="account">
							<li><a id="loginLink" href="">Login</a></li>
				<li><a href="accounts/register">Register</a></li>
					</ul>
	</header>

	<div id="main">
		
		<header>
			<h1></h1>
		</header>

		<div id="content">
			woo
				<div class="intro">
		
	</div>
		</div>
	</div>

	<script type="text/javascript" src="/resources/lib/jquery-1.8.2.min.js"></script>
	<script type="text/javascript" src="/resources/lib/packery.pkgd.min.js"></script>
	<script type="text/javascript" src="/resources/js/ontherocks.js"></script>
</body>
</html>
